build.gradle

```diff
@@ -73,4 +76,33 @@ sonar {
 
 tasks.named('sonar').configure {
 	dependsOn jacocoTestReport
+}
+
+spotless {
+	java {
+
+		importOrder()
+		toggleOffOn()
+		palantirJavaFormat()
+		removeUnusedImports()
+		trimTrailingWhitespace()
+		endWithNewline()
+
+		cleanthat()
+		googleJavaFormat()
+
+		formatAnnotations()
+
+		licenseHeader '/* (C) 2023 */'
+
+	}
+
+	groovy {
```

Comment:
we can simplify groovy conf. using the eclipse format:
https://github.com/diffplug/spotless/blob/main/plugin-gradle/README.md#eclipse-groovy

p.s: using only default, the optional conf. is not necessary